fiftyone/server/color.py

```diff
@@ -88,6 +98,21 @@ def run():
             dataset.app_config.color_scheme = (
                 foo.ColorScheme(
                     color_pool=color_scheme.color_pool,
+                    color_by=color_scheme.color_by
```

Comment:
Is there an issue with providing these values without a truthy check? 
```suggestion
                    color_by=color_scheme.color_by,
```